src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -133,6 +133,17 @@ public override void Load()
                 throw;
             }
 
+            // Set the cache expiration time for all refresh registered settings
+            foreach (KeyValueWatcher changeWatcher in _options.ChangeWatchers)
+            {
+                changeWatcher.CacheExpires = DateTimeOffset.UtcNow.Add(changeWatcher.CacheExpirationInterval);

```

Comment:
>DateTimeOffset.UtcNow [](start = 45, length = 21)

Can we call `DateTimeOffset.UtcNow` once instead of one call per change watcher?